### PR TITLE
Tickets/dm 28039

### PR DESCRIPTION
**Pull request by gcomoretto**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -82,10 +82,12 @@ requirements:
     - gsl
     - h5py
     - healpy
+    - healsparse
     - lmfit
     - lsstdesc.coord
     - matplotlib-base
-    - moto
+    - mock =4.0.2
```
Comment (timj) [inline recipe/meta.yaml:89]: Have they not fixed this problem yet?

**Review by timj — APPROVED** [review]
It's fine. I really don't understand why a month has gone by and that mock/moto problem hasn't been fixed by upstream

File: recipe/meta.yaml (hunk 1 above)
Comment (gcomoretto) [reply recipe/meta.yaml:89]: resolving the rubin-env version 0.1.4 I still see mock 4.0.3 and moto 1.3.16, which give us the problem.

**Comment by gcomoretto** [comment]
@conda-forge-admin, please rerender

**Comment by gcomoretto** [comment]
@beckermr, can I move forward with the merge, despite the problem in the rerender?

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Pull request merged**